flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "skip-icons-check": true,
+    "automerge-flathubbot-prs": true,
+    "only-arches": ["x86_64"]
```

Comment:
why x86_64 only?

Reply:
Well, mostly cause I copied it from the podman PR. But I intentionally left it as is as this was my first flatpak build and I wanted to minimize complexity.

Reply:
I meant you should remove this, there is no reason

Reply:
Fixed: https://github.com/flathub/flathub/pull/4119/commits/f5c1f59bc488a206214668d50af391ef99bd8037